src/views/SpaceCreate.vue

```diff
@@ -396,29 +416,68 @@ const needsPluginConfigs = computed(() =>
-              class="mb-2 w-full break-all"
-            />
-            <input
-              v-else
-              v-model="form.name"
-              maxlength="128"
-              class="text-2xl font-semibold input mb-2 w-full"
-              :placeholder="$t('create.question')"
-              ref="nameInput"
+              class="w-full break-all"
             />
-            <div class="relative group">
-              <TextareaAutosize
-                v-if="!preview"
+            <SBase v-else :definition="{ title: $t('create.proposalTitle') }">
+              <input
+                v-model="form.name"
+                maxlength="128"
+                class="text-lg font-semibold s-input w-full !rounded-[12px]"
+                ref="nameInput"
+              />
+            </SBase>
+
+            <div v-if="!preview">
+              <div class="flex justify-between">
+                <label
+                  v-text="$t('create.proposalDescription')"
+                  class="s-label"
+                />
+                <div class="text-xs">
+                  {{ form.body.length }}/{{ bodyLimit }}
+                </div>
+              </div>
+              <textarea
+                ref="textAreaEl"
+                class="s-input pt-0 w-full !rounded-t-[12px] !rounded-b-none min-h-[50vh]"
+                :maxLength="bodyLimit"
                 v-model="form.body"
-                class="input pt-0 w-full"
-                style="font-size: 22px"
-                :placeholder="$t('create.content')"
-                :max-length="bodyLimit"
               />
+              <label
+                class="relative flex justify-between bg-skin-border rounded-b-[12px] py-1 px-2 items-center"
+              >
+                <input
+                  accept="image/jpg, image/jpeg, image/png"
+                  type="file"
+                  class="opacity-[0.001] absolute p-[5px] cursor-pointer top-0 right-0 bottom-0 left-0 w-full ml-0"
+                  @change="upload"
+                />
+
+                <span class="pointer-events-none relative pl-1 text-sm">
+                  <span v-if="uploading"
+                    ><LoadingSpinner class="inline" />
+                    {{ $t('create.uploading') }}</span
+                  >
+                  <span v-else-if="imageUploadError !== ''">
+                    {{ imageUploadError }}</span
+                  >
+                  <span v-else>
+                    {{ $t('create.uploadImageExplainer') }}
+                  </span>
+                </span>
+                <a
+                  class="relative inline"
+                  href="https://docs.github.com/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax"
+                  target="_blank"
+                  v-tippy="{ content: $t('create.markdown') }"
+                >
+                  <BaseIcon name="github" class="text-skin-text" />
```

Comment:
Do you need me to add the markdown icon?

Reply:
Yes.